doc/merkle_ZH-CN.md

```diff
@@ -0,0 +1,128 @@
+# Merkle Structures
+
+*阅读其它语言版本: [English](merkle.md), [Korean](merkle_KR.md).*
+
+MimbleWimble 是设计给用户在仅给出修剪后的数据的情况下也能验证系统状态。
```

Comment:
Could you replace all instances of `MimbleWimble` by Mimblewimble` (in this PR) there is #3118 for the rest.